modules/editorial-metadata/editorial-metadata.php

```diff
@@ -136,6 +156,20 @@ public function action_admin_enqueue_scripts() {
 		}
 	}
 
+	public function load_scripts_for_block_editor() {
+		$asset_file = include VIP_WORKFLOW_ROOT . '/dist/modules/editorial-metadata/editorial-metadata-block.asset.php';
+		wp_enqueue_script( 'vip-workflow-block-editorial-metadata-script', VIP_WORKFLOW_URL . 'dist/modules/editorial-metadata/editorial-metadata-block.js', $asset_file['dependencies'], $asset_file['version'], true );
+
+		$editorial_metadata_terms = $this->get_editorial_metadata_terms();
+		wp_localize_script( 'vip-workflow-block-editorial-metadata-script', 'VipWorkflowEditorialMetadatas', $editorial_metadata_terms );
```

Comment:
I've noticed we use a variety of styles for the localized global (e.g. `VW_EDITORIAL_METADATA_CONFIGURE` or `VipWorkflowEditorialMetadatas`), and also sometimes passing an array directly or an array of values.

I'd personally prefer we use the caps-underscore style, and pass any values as a named key, like:


```suggestion
		wp_localize_script( 'vip-workflow-block-editorial-metadata-script', 'VW_EDITORIAL_METADATA', [
		    'terms' => $editorial_metadata_terms
		] );
```

The named-key style just makes it easier to pass another bit of data into the page without affecting how the terms are being consumed.

Reply:
Agreed on  that, I'll make that change